includes/Dashboard.php

```diff
@@ -261,7 +261,7 @@ public function get_dashboard_settings() {
 				'version'      => WEBSTORIES_VERSION,
 				'api'          => [
 					'stories'   => sprintf( '/web-stories/v1/%s', $rest_base ),
-					'media'     => '/web-stories/v1/media',
+					'media'     => '/wp/v2/media',
```

Comment:
Why was this changed? 🤔 

`/web-stories/v1/media` should be correct. It is the same as `/wp/v2/media` but also supports `?=_web_stories_envelope=1`

Reply:
ah, i changed it because of this: https://github.com/google/web-stories-wp/issues/3032#issuecomment-663494150 

I will swap it back!

Reply:
Swapped and confirmed. 

<img width="982" alt="Screen Shot 2020-08-18 at 8 29 25 AM" src="https://user-images.githubusercontent.com/10720454/90533307-2a704480-e12d-11ea-92ed-fe333c069f96.png">
<img width="552" alt="Screen Shot 2020-08-18 at 8 29 39 AM" src="https://user-images.githubusercontent.com/10720454/90533310-2ba17180-e12d-11ea-843b-eaa9589051fd.png">